estate/models/estate_property.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from typing_extensions import Required
+from odoo import fields, models
+
+
+class EstateProperty(models.Model):
+    _name = "estate_property"
```

Comment:
Name should be with dots, not underscores (the dots will be converted to underscores in the db)